common.mk

```diff
@@ -64,7 +66,7 @@ $(VERSIONS): % : %/root/help.1
 .PHONY: test check
 check: test
 
-test: script_env += TEST_MODE=true
+test: script_env += TEST_MODE=local
```

Comment:
The strange thing is, that this variable now has different meaning...

Previously it was used to switch the test suite `on`, not to define test `variant` (or environment). I'd say the variable chosen before was somehow unfortunate.

Reply:
do I invent a new one then?

Reply:
There is some meaning change behind the variable usage but I do not think it is necessary to introduce a different one just because of that. The variable should only be used in common scripts anyway.

Reply:
The issue I see with the way it is used now is that there is no way to set this variable to any other value.